core/claimmanager.go

```diff
@@ -20,6 +22,8 @@ import (
 var ErrClaim = errors.New("ErrClaim")
 var ErrClaimManager = errors.New("ErrClaimManager")
 
+const DefaultClaimRetryInterval = time.Second * 5
```

Comment:
What's the circumstance that a claim would fail, in which it would work 5 seconds later? The geth node not running and then running again? Is there a timeout for the transaction to get confirmed? 

Should it be at least 15 seconds so that we wait for the next block, or does this have nothing to do with it getting included in a block?

Reply:
I think the main scenario where a claim would fail (excluding bad parameters) is if you already have a pending transaction (you might've just called `reward` or `initializeRound`) and the nonce for the claim transaction is the same as the nonce for your pending transaction so geth thinks you are trying to replace the pending transaction with the claim transaction, but rejects the claim transaction because the gas price is too low for replacement. Issue described here: https://github.com/livepeer/go-livepeer/issues/169 

I think setting the retry interval to the block time would work since after the retry interval any pending transaction you have should be mined and your nonce should be updated properly. But, it doesn't seem ideal because you still can't have multiple pending transactions at once. Might be worth looking into a fix for the issue mentioned above because that would fix failing valid claim transactions and also allow you to have multiple pending transactions at once